test/fixtures/react-native/pressability/expected.js

```diff
@@ -775,8 +775,9 @@ export default class Pressability {
       nativeTag = this._responderID._nativeTag;
     }if (nativeTag == null) {
       return;
-    }
-    const { onLongPress, onPress } = this._config;var hasPress = !!onPress;var hasLongPress = !!onLongPress;UIManager.onFsPressForward(nativeTag, isLongPress, hasPress, hasLongPress);
+    }const {
+      onLongPress,
+      onPress } = this._config;var hasPress = !!onPress;var hasLongPress = !!onLongPress;UIManager.onFsPressForward(nativeTag, isLongPress, hasPress, hasLongPress);
```

Comment:
Did anything besides formatting actually change in this section?

Maybe I should set up something to format the code automatically on pre-commit...

Reply:
Just formatting, presumably from some updated babel dependencies.

Reply:
Update: I added this to my plate for next sprint: https://app.shortcut.com/fullstory/story/167777/set-up-automatic-code-formatting-and-linting-for-react-native-babel-libraries